### PR TITLE
Fixing sql fields

### PR DESCRIPTION
**Pull request by Bakual**
Pull Request for Issue #13088.

### Summary of Changes
* Adds PostgreSQL commands that were missing in previous PR
* Merges update commands related to com_fields into one file.

### Testing Instructions
* Test an update from 3.6.4 to staging with MySQL and/or PostgreSQL and verify that fields and firleds group managing works.
* Also test a fresh install of staging

### Documentation Changes Required
None

#### Notes
Once this is merged, fields still need SQL Azure SQL commands.

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql [hunk 1]
```diff
@@ -42,6 +42,34 @@ CREATE INDEX "#__fields_idx_context" ON "#__fields" ("context");
...
+  "ordering" bigint DEFAULT 0 NOT NULL,
+  "language" varchar(7) DEFAULT '' NOT NULL,
+  "created" timestamp without time zone DEFAULT '1970-01-01 00:00:00' NOT NULL,
+  "created_by" bigint DEFAULT 0 NOT NULL,
+  "modified" timestamp without time zone DEFAULT '1970-01-01 00:00:00' NOT NULL,
+  "modified_by" bigint DEFAULT 0 NOT NULL,
+  "access" bigint DEFAULT 0 NOT NULL,
+  PRIMARY KEY ("id")
+);
+CREATE INDEX "#__fields_idx_checked_out" ON "#__fields_groups" ("checked_out");
```
Comment (alikon) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql]: `sql CREATE INDEX "#__fields_idx_checked_out" ON "#__fields_groups" ("checked_out");`

cannot see a `checked_out` field on` #__fields_group` table declaration

Comment (alikon) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql]: seems missed from mysql table declaration

**Review by alikon — COMMENTED** [review]
need some small fix for postgresql  the `#__fields_groups` lacks 2 fields than mysql `checked_out` `checked_out_time`

File: administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql (hunk 1 above)
Comment (Bakual) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql]: Indeed missed those on the update files. Do you want to do the PR or should I do it?

Comment (alikon) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql]: isn't better to revert+fix+remerge ?

Comment (Bakual) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql]: Nah, we just can add those two columns in a different PR.

Comment (alikon) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2016-08-29.sql]: ok see #13102

**Pull request merged**